### DISCUSSION TITLE
A compiler error

### DISCUSSION THREAD

**Discussion by xiuer-1573** [alvr-org/ALVR#1017] [Q&A]
when i use cargo xtask build-server,The following error information is displayed
PS C:\Users\Administrator\Desktop\ALVR-18.2.1> cargo xtask build-server
    Updating git repository `https://github.com/zarik5/android-ndk-rs`
error: failed to get `ndk` as a dependency of package `client v17.0.0-dev.1 (C:\Users\Administrator\Desktop\ALVR-18.2.1\alvr\experiments\client)`

Caused by:
  failed to load source for dependency `ndk`

Caused by:
  Unable to update https://github.com/zarik5/android-ndk-rs?rev=0a547b5ab2d698d80bde1b8ebb6dfe06d550baff#0a547b5a

Caused by:
  object not found - no match for id (0a547b5ab2d698d80bde1b8ebb6dfe06d550baff); class=Odb (9); code=NotFound (-3)

Git update failed. What should I do?
thank you!

**Comment by zmerp** [comment]
That's very strange, I don't get the error, even after running `cargo clean`. In any case I pushed a fix in the v18 branch (the fix was already there in the master branch).
